### PR TITLE
[Docs] Update support/grafana docs

### PR DESCRIPTION
**Pull request by sgibson91**
This PR updates our support docs after #1047

### CODE REVIEW

File: docs/howto/operate/grafana.md [hunk 1]
```diff
@@ -23,22 +23,33 @@ This guide will walk through the steps required to setup a suite of Grafana dash
...
 These tools are [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/), for controlling ingresses and load balancing; [`cert-manager`](https://cert-manager.io/docs/), for automatically provisioning TLS certificates from [Let's Encrypt](https://letsencrypt.org/); [Prometheus](https://prometheus.io/), for scraping and storing metrics from the cluster and hub; and [Grafana](https://grafana.com/), for visualising the metrics retreived by Prometheus.
 
-#### Edit your `cluster.yaml` file
+#### Create a `support.values.yaml` file in your chosen cluster folder
+
+In the `infrastructure` repo, the full filepath should be: `config/clusters/<cluster_name>/support.values.yaml`.
 
-Add the following config as a top-level key to your `cluster.yaml` file under `/config/clusters/<cluster_name>` in `infrastructure`.
+Add the following helm chart values to your `support.values.yaml` file.
 `GRAFANA_URL` should follow the pattern `grafana.<cluster_name>.2i2c.cloud`.
```
Comment (consideRatio) [inline docs/howto/operate/grafana.md:31]: Where is GRAFANA_URL to be configured etc?

Comment (sgibson91) [reply docs/howto/operate/grafana.md:31]: Yes, it should be `<grafana_domain>`

Comment (sgibson91) [inline docs/howto/operate/grafana.md:31]:
```suggestion
`<grafana-domain>` should follow the pattern `grafana.<cluster_name>.2i2c.cloud`.
```

**Review by consideRatio — APPROVED** [review]
🎉🎉🎉🎉🎉

**Review by GeorgianaElena — APPROVED** [review]
:rocket:

**Pull request merged**